test/unit/views/HowManyFilesUploadSpec.scala

```diff
@@ -57,7 +57,18 @@ class HowManyFilesUploadSpec extends DomAssertions with IntViewBehaviours[FileUp
     behave like intPage(createAppendable, (form, i) => form.bind(Map("value" -> i.toString)), "value", messagePrefix)
 
     "display the correct guidance" in {
-      val expectedGuidanceKeys: List[String] = List("paragraph1", "paragraph2", "paragraph3", "listItem1", "listItem2", "listItem3", "listItem4")
+      val expectedGuidanceKeys: List[String] = List(
+        "paragraph1",
+        "paragraph2.heading",
+        "paragraph2",
+        "paragraph3.heading",
+        "paragraph3",
+        "listItem1",
+        "listItem2",
+        "listItem3",
+        "listItem4",
+        "warning"
+      )
       for (key <- expectedGuidanceKeys) assertContainsText(createView(), messages(s"$messagePrefix.$key"))
```

Comment:
Re-creating the view at every iteration seems actually superfluous. What about assigning the result from `createView` to a `val`?

Reply:
Good spot, altered :+1: